docs/cookbook/extending-product-list.md

```diff
@@ -135,7 +135,43 @@ $structure = \array_merge($structure, [
 ]);
 ```
 
-### 4. Extend `ListedProductViewFactory` so it returns the new required data
+### 4. Extend `ProductElasticsearchConverter` to fill empty fields
+
+There are old documents in the Elasticsearch, usually in the production environment.
+Before you reexport all products from the database, there are documents that don't have the new field `brand_name`.
+So you have to provide default values for the case of reading such old documents.
+
+```php
+declare(strict_types=1);
+
+namespace Shopsys\ShopBundle\Model\Product\Search;
+
+use Shopsys\FrameworkBundle\Model\Product\Search\ProductElasticsearchConverter as BaseProductElasticsearchConverter;
+
+class ProductElasticsearchConverter extends BaseProductElasticsearchConverter
```

Comment:
Nice catch!

And now, I'll catch you! You have to fix the test only during development in monorepo because this test is in the framework. When you develop your project, you don't run framework tests...
Or do you?! :thinking: 

But even if you run FW tests in the project, the test will pass, because the test is unit and create the instance by itself. So if you extend the class, the test isn't affected.